wagtail/contrib/forms/bulk_actions/form_bulk_action.py

```diff
@@ -0,0 +1,17 @@
+from wagtail.admin.views.bulk_action import BulkAction
+from django.utils.translation import gettext_lazy as _
+from wagtail.contrib.forms.models import FormSubmission
+
+
+class FormBulkAction(BulkAction):
```

Comment:
It would be good to use consistent naming - refer to form submission not form.

Reply:
I tried to come up with names, I will name it as FormSubmission.

Reply:
When naming things I find it's helpful to write a docstring first or look at the conventions set by similar things. 

For example - this is the BulkActions class for the FormSubmission

Then you can name it from there and remove the docstring. 

FormSubmissionBulkAction. 

Also it would be good to rename the file.